docs/man/mame.6

```diff
@@ -927,9 +927,9 @@ Sets the startup volume. It can later be changed with the user interface
 .TP
 .B \-audio_latency \fIvalue
 This controls the amount of latency built into the audio streaming.
-The latency parameter controls the lower threshold. The default is 1
-(meaning lower=1/5 and upper=2/5). Set it to 2 (\-audio_latency 2) to keep
-the sound buffer between 2/5 and 3/5 full. If you crank it up to 4,
+The latency parameter controls the lower threshold. The default is 2
+(meaning lower=2/5 and upper=3/5). Set it to 1 (\-audio_latency 1) to keep
+the sound buffer between 1/5 and 2/5 full. If you crank it up to 4,
```

Comment:
The behaviour of this option depends on the audio output module used.  This description doesn't really make sense when I read it, and doesn't describe what the option does for CoreAudio or SDL output.